### PR TITLE
change logging statements for renter

### PR DESCRIPTION
**Pull request by DavidVorick**
In debug mode, the renter will log the number of contracts it found for a given file. Seemed like it might help in some places.

In normal mode, the renter will return an error if the download does not get enough pieces, instead of if it can find zero hosts. I think this was  the desired behavior originally.

### CODE REVIEW

File: modules/renter/download.go [hunk 1]
```diff
@@ -204,6 +204,7 @@ func (r *Renter) Download(path, destination string) error {
 			contracts[&fc] = c
 		}
 	}
+	r.log.Debugf("Starting Download, found %v contracts\n", len(contracts))
```
Comment (lukechampine) [inline modules/renter/download.go]: shouldn't be done inside the lock

File: modules/renter/download.go [hunk 2]
```diff
@@ -223,8 +224,8 @@ func (r *Renter) Download(path, destination string) error {
 		defer d.Close()
 		hosts = append(hosts, newHostFetcher(d, *fc, file.masterKey))
 	}
-	if len(hosts) == 0 {
-		return errors.New("Could not connect to any hosts:\n" + strings.Join(errs, "\n"))
+	if len(hosts) < file.erasureCode.MinPieces() {
+		return errors.New("Could not connect to enough hosts:\n" + strings.Join(errs, "\n"))
```
Comment (lukechampine) [inline modules/renter/download.go]: whoops, yeah forgot to revert this once we had more hosts

File: modules/renter/download.go (hunk 1 above)
Comment (DavidVorick) [inline modules/renter/download.go]: I thought you needed to use the logger in a thread-safe way?

Comment (lukechampine) [inline modules/renter/download.go]: nah, loggers guarantee serialized access

**Pull request merged**